test/bigtest/mocks/form-config-samples.js

```diff
@@ -682,15 +682,16 @@ export const formConfigSamples = [{
               childControls: [{
                 controlType: 'Col',
                 staticControlType: 'Col',
-                xs: 6,
+                xs: 4,
                 dataAttributes: { 'data-test-suppress-from-discovery': '' },
                 childControls: [{
                   controlType: 'Field',
                   staticControlType: 'KeyValue',
                   component: 'Checkbox',
                   label: 'ui-data-import.settings.mappingProfiles.map.administrativeData.field.discoverySuppress',
-                  name: 'mappingDetails.mappingFields[0].value',
-                  enabled: true,
```

Comment:
Are you sure this field is not needed?

Reply:
Sure, it was changed to name: 'mappingDetails.mappingFields[0].booleanFieldAction',

Reply:
No, I mean `enabled` field

Reply:
it's not needed here any more, cause here always should be  a value from select